### PR TITLE
Markdown changes

### PR DESCRIPTION
**Pull request by domitechnshimba**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?
- [x] Have you successfully run `brew man` locally and committed any changes?

-----

### CODE REVIEW

File: docs/Gems,-Eggs-and-Perl-Modules.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ chmod +a 'user:<YOUR_NAME_HERE> allow add_subdirectory,add_file,delete_child,dir
 will let you add packages to Python 3.y as yourself, which
 is probably safer than changing the group ownership of the directory.
 
-### So why was I using sudo?
+### So why was I using sudo
```
Comment (reitermarkus) [inline docs/Gems,-Eggs-and-Perl-Modules.md:22]: Re-add the question mark.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by reitermarkus** [comment]
@domitechnshimba, please squash this into one commit with a more descriptive commit message.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 2]
```diff
@@ -5,69 +5,96 @@ The following commands are used by Homebrew contributors to set up a fork of Hom
...
     ```
+
 3. Add your pushable forked repository as a new remote:
+
     ```sh
     git remote add <YOUR_USERNAME> https://github.com/<YOUR_USERNAME>/brew.git
     ```
-  * `<YOUR_USERNAME>` is your GitHub username, not your local machine username.
+
+* `<YOUR_USERNAME>` is your GitHub username, not your local machine username.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:31]: This should stay in `3.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 3]
```diff
@@ -5,69 +5,96 @@ The following commands are used by Homebrew contributors to set up a fork of Hom
...
     ```
+
 3. Add your pushable forked repository as a new remote:
+
     ```sh
     git remote add <YOUR_USERNAME> https://github.com/<YOUR_USERNAME>/homebrew-core.git
     ```
-  * `<YOUR_USERNAME>` is your GitHub username, not your local machine username.
+
+* `<YOUR_USERNAME>` is your GitHub username, not your local machine username.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:51]: This should stay in `3.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 4]
```diff
@@ -5,69 +5,96 @@ The following commands are used by Homebrew contributors to set up a fork of Hom
...
 
 ### Formulae related pull request
 
 1. [Fork the Homebrew/homebrew-core repository on GitHub](https://github.com/Homebrew/homebrew-core/fork).
-  * This creates a personal remote repository that you can push to. This is needed because only Homebrew maintainers have push access to the main repositories.
+
+* This creates a personal remote repository that you can push to. This is needed because only Homebrew maintainers have push access to the main repositories.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:37]: This should stay in `1.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 5]
```diff
@@ -5,69 +5,96 @@ The following commands are used by Homebrew contributors to set up a fork of Hom
...
 3. Create a new branch from the latest `master` branch:
+
     ```sh
     git checkout -b <YOUR_BRANCH_NAME> origin/master
     ```
+
 4. Make your changes. For formulae, use `brew edit` or your favourite text editor, following all the guidelines in the [Formula Cookbook](Formula-Cookbook.md).
-  * If there's a `bottle do` block in the formula, don't remove or change it; we'll update it when we pull your PR.
+
+* If there's a `bottle do` block in the formula, don't remove or change it; we'll update it when we pull your PR.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:77]: This should stay in `4.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 6]
```diff
@@ -5,69 +5,96 @@ The following commands are used by Homebrew contributors to set up a fork of Hom
...
     brew tests
     brew install --build-from-source <CHANGED_FORMULA>
     brew test <CHANGED_FORMULA>
     brew audit --strict <CHANGED_FORMULA>
     ```
+
 6. [Make a separate commit](Formula-Cookbook.md#commit) for each changed formula with `git add` and `git commit`.
-  * Please note that our preferred commit message format for simple version updates is "`<FORMULA_NAME> <NEW_VERSION>`", e.g. "`source-highlight 3.1.8`".
+
+* Please note that our preferred commit message format for simple version updates is "`<FORMULA_NAME> <NEW_VERSION>`", e.g. "`source-highlight 3.1.8`".
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:90]: This should stay in `6.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 7]
```diff
@@ -79,24 +106,32 @@ To respond well to feedback:
 
 1. Ask for clarification of anything you don't understand and for help with anything you don't know how to do.
 2. Post a comment on your pull request if you've provided all the requested changes/information and it hasn't been merged after a week. Post a comment on your pull request if you're stuck and need help.
-  * A `needs response` label on a PR means that the Homebrew maintainers need you to respond to previous comments.
+
+* A `needs response` label on a PR means that the Homebrew maintainers need you to respond to previous comments.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:110]: This should stay in `2.`.

File: docs/How-To-Open-a-Homebrew-Pull-Request.md [hunk 8]
```diff
@@ -79,24 +106,32 @@ To respond well to feedback:
...
+
 2. Make any requested changes and commit them with `git add` and `git commit`.
 3. Squash new commits into one commit per formula:
+
     ```sh
     git rebase --interactive origin/master
     ```
-  * If you are working on a PR for a single formula, `git commit --amend` is a convenient way of keeping your commits squashed as you go.
+
+* If you are working on a PR for a single formula, `git commit --amend` is a convenient way of keeping your commits squashed as you go.
```
Comment (reitermarkus) [inline docs/How-To-Open-a-Homebrew-Pull-Request.md:131]: This should stay in `3.`.

File: docs/Releases.md [hunk 9]
```diff
@@ -1,44 +1,22 @@
...
+2. Write up a release notes blog post to <https://brew.sh> e.g. [brew.sh#319](https://github.com/Homebrew/brew.sh/pull/319). This should use `brew release-notes` as input but have the wording adjusted to be more human readable and explain not just what has changed but why.
+3. When the release has shipped and the blog post has been merged, tweet the blog post as the [@MacHomebrew Twitter account](https://twitter.com/MacHomebrew) or tweet it yourself and retweet it with the @MacHomebrew Twitter account (credentials are in 1Password).
+4. Send the email to the Homebrew TinyLetter email list (credentials are in 1Password).
 5. Consider whether to submit it to other sources e.g. Hacker News, Reddit.
 
-  - Pros: gets a wider reach and user feedback
-  - Cons: negative comments are common and people take this as a chance to
-          complain about Homebrew (regardless of their usage)
+- Pros: gets a wider reach and user feedback
+- Cons: negative comments are common and people take this as a chance to complain about Homebrew (regardless of their usage)
```
Comment (reitermarkus) [inline docs/Releases.md:22]: These should stay in `5.`.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: docs/Gems,-Eggs-and-Perl-Modules.md (hunk 1 above)
Comment (domitechnshimba) [reply docs/Gems,-Eggs-and-Perl-Modules.md]: I have added a question mark

**Comment by MikeMcQuaid** [comment]
Passing on this, sorry!

**Pull request closed without merging**